drivers/scale_service.go

```diff
@@ -66,6 +66,9 @@ func (s *ScaleServiceDriver) Execute(conf interface{}, apiClient client.RancherC
 	if scaleAction == "up" {
 		newScale = service.Scale + int64(scaleChange)
 	} else if scaleAction == "down" {
+		if service.Scale-int64(scaleChange) < 0 {
```

Comment:
@cjellick changed, not added any tests yet, as we discussed I'll move to Prometheus first and then when the fields for min/max are added, I can add detailed tests